examples/nodejs/token-vending-machine/lambda/authorizer/authorizer.ts

```diff
@@ -0,0 +1,35 @@
+import { APIGatewayAuthorizerResult, APIGatewayRequestAuthorizerEvent } from 'aws-lambda';
+
+// Lambda function for API Gateway Lambda Authorizer
+// Input format: https://docs.aws.amazon.com/apigateway/latest/developerguide/api-gateway-lambda-authorizer-input.html 
+// Output format: https://docs.aws.amazon.com/apigateway/latest/developerguide/api-gateway-lambda-authorizer-output.html 
+export const handler = async (event: APIGatewayRequestAuthorizerEvent): Promise<APIGatewayAuthorizerResult> => {
```

Comment:
in Lambda code it's a good idea to add a few `console.log` statements to help the user debug if something goes wrong. I would probably add a log statement indicating whether the headers were found or not, one that indicates "Allow", and one that gives a hint as to why we were denied if we are denied.

you can see these log outputs in the AWS lambda console or in the AWS CloudWatch console after your lambda executes.